app/controllers/course/courses_controller.rb

```diff
@@ -2,6 +2,8 @@
 class Course::CoursesController < Course::Controller
   include Course::ActivityFeedsConcern
 
+  skip_before_action :authenticate_user!, only: :index
```

Comment:
`ApplicationUserConcern` has some way to skip `authenticate_user!` in https://github.com/Coursemology/coursemology2/blob/master/app/controllers/concerns/application_user_concern.rb#L11, depending on `publicly_accessible?`

I'm not sure, but maybe it's possible to make use of this to skip the auth check by overriding `publicly_accessible?` somewhere?